tests/test_main.py

```diff
@@ -708,6 +711,9 @@ def test_module_files(self):
             if (
                     # The sentinel actually appearing in code messes things up
                     'executing' in filename
+                    # ignore pytest files because they break the SentinelNodeFinder
+                    or '_pytest' in filename
+                    or 'enum' in filename
```

Comment:
This file seems to only be a problem in pypy, as they seem to have had deadcode analysis for much longer than CPython. Can you move this case inside `or PYPY and ('sysconfig.py' in filename or 'pyparsing.py' in filename)` just below?

Reply:
I changed it.